draft-ietf-tls-tls13.md

```diff
@@ -1646,7 +1659,8 @@ like that of C.
        } T;
 
 Fixed- and variable-length vector fields are allowed using the standard vector
-syntax. Structures V1 and V2 in the variants example below demonstrate this.
+syntax (but arrays of either sort must themselves only contain fixed-length
+elements). Structures V1 and V2 in the variants example below demonstrate this.
```

Comment:
Not entirely clear that we actually need to say this, since we do call it an informal presentation language.  But I believe we need a fixed per-element size in order to do arrays, and don't think we say that explicitly anywhere.

Reply:
This is not the case.  You can have an array of variable-length elements.  Those elements necessarily include length prefixes.  What you can't have is arrays of things that do not have a means by which the length is known.

See for example from ClientHello:
```
           Extension extensions<8..2^16-1>;
```

Reply:
Wow, massive braino, sorry about that.  Presumably I was thinking about C arrays or something